storage/google/cloud/storage/client.py

```diff
@@ -372,11 +396,47 @@ def create_bucket(self, bucket_or_name, requester_pays=None, project=None):
             >>> bucket = client.create_bucket(bucket)  # API request.
 
         """
+        if user_project is not None:
+            raise ValueError("Cannot create bucket with 'user_project' set.")
```

Comment:
This shouldn't be the case. You should be able to create a bucket with user_project set, now at least. It might be technical debt from Requester Pays feature implementation.

Reply:
As I understand, buckets/insert only uses `user_project` as query parameter, so we don't do `bucket.user_project=user_project` while creating new bucket.

Coverage failed, I assume, because of skips. I've only moved two lines with last commit, this change can't be the cause of 10 coverage issues, I believe.